pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -638,9 +637,6 @@ protected void setupCustomRouting(@NonNull Config cfg) {
         AuthenticationRoute authenticationRoute = new AuthenticationRoute(auth0Util,
                 userUtil,
                 cfg.getString(ApplicationConfigConstants.AUTH0_DOMAIN),
-                cfg.getString(ApplicationConfigConstants.AUTH0_MGT_SECRET),
-                cfg.getString(ApplicationConfigConstants.AUTH0_MGT_KEY),
-                cfg.getString(ApplicationConfigConstants.AUTH0_MGT_API_URL),
                 cfg.getString(ApplicationConfigConstants.AUTH0_CLAIM_NAMESPACE)
         );
```

Comment:
Had to refactor and fix Auth0Util access was being done. Refactoring ended up requiring modifying callers. More on these changes below.